ui/app/scripts/routes.js

```diff
@@ -77,13 +85,21 @@ define(['./app'], function (xdAdmin) {
       }
     })
     .state('home.jobs.tabs', {
-      url : 'tasks',
+      url : 'jobs',
```

Comment:
Why `jobs` here? Also, I am confused by the existence of `job` based routes (vs `task` based ones) in this file.